examples/src/basic/mod.rs

```diff
@@ -1,3 +1,4 @@
 pub mod cron;
 pub mod endpoint;
+pub mod url;
```

Comment:
@quesabe I would only keep endpoint, worker, and cron as basic endpoints. Let's move this to `src/urls.rs`.